src/main/java/org/apache/commons/lang3/reflect/MemberUtils.java

```diff
@@ -162,19 +165,23 @@ private static float getTotalTransformationCost(final Class<?>[] srcArgs, final
             // When isVarArgs is true, srcArgs and dstArgs may differ in length.
             // There are two special cases to consider:
             final boolean noVarArgsPassed = srcArgs.length < destArgs.length;
+            @SuppressWarnings("index:array.access.unsafe.low") // isVarArgs => srcArgs.length != 0
```

Comment:
`isVarArgs` could also be annotated with `@EnsuresMinLenIf`

Reply:
But which variable's minimum length would it ensure? I'm unable to figure it out.

Reply:
My comment here was in error. I thought that `isVarArgs` was a method that was called on `srcArgs` for some reason, which it clearly isn't. You can disregard this comment.